doc/guide/installation.rst

```diff
@@ -16,6 +17,11 @@ The easiest way to keep your suite updated is to use
     .. code-block:: js
 
         {
+            "require": {
+                "behat/behat": "~3.0",
+                "behat/mink-extension": "~2.0@dev",
+                "behat/mink-goutte-driver": "*"
```

Comment:
I've added it because [it's a dev requirement]((https://packagist.org/packages/behat/mink-extension)) for `behat/mink-extension`. Also because it would eventually not work out of the box without any driver (or indication that a driver would be needed).